From e290be3134994f8b84c9d0026666d4f146131d86 Mon Sep 17 00:00:00 2001
From: D'Orus Tsitera <dorustsitera@gmail.com>
Date: Mon, 13 Jul 2026 19:14:47 +0000
Subject: [PATCH 04481/15333] Documentation: PCI: Fix sysfs-bus-pci typo

Replace "similiar" with "similar" in
Documentation/ABI/testing/sysfs-bus-pci.

Signed-off-by: D'Orus Tsitera <dorustsitera@gmail.com>
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Link: https://patch.msgid.link/20260713-docs-fix-similiar-v1-1-257e25150346@gmail.com
---
 Documentation/ABI/testing/sysfs-bus-pci | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/ABI/testing/sysfs-bus-pci b/Documentation/ABI/testing/sysfs-bus-pci
index b767db2c52cb..55ea1db749a1 100644
--- a/Documentation/ABI/testing/sysfs-bus-pci
+++ b/Documentation/ABI/testing/sysfs-bus-pci
@@ -171,7 +171,7 @@ Description:
 		all devices attached through the subordinate bus of a specific
 		bridge device, writing 1 to this will try to do it.  This will
 		affect all devices attached to the system through this bridge
-		similiar to writing 1 to their individual "reset" file, so use
+		similar to writing 1 to their individual "reset" file, so use
 		with caution.
 
 What:		/sys/bus/pci/devices/.../vpd
-- 
2.55.0

